manim/scene/scene.py

```diff
@@ -373,6 +378,17 @@ def add_mobjects_among(self, values):
         self.add(*filter(lambda m: isinstance(m, Mobject), values))
         return self
 
+    def add_mobjects_from_animations(self, animations):
+
```

Comment:
Question: why the blank line? Did black do this? Seems odd.

Reply:
I don't know, but black seems to be okay with it. Should I remove it ?